examples/run_glue_tpu.py

```diff
@@ -342,14 +342,17 @@ def main(args):
     config = config_class.from_pretrained(args.config_name if args.config_name else args.model_name_or_path,
                                           num_labels=num_labels,
                                           finetuning_task=args.task_name,
-                                          cache_dir=args.cache_dir if args.cache_dir else None)
+                                          cache_dir=args.cache_dir if args.cache_dir else None,
+                                          xla_device=True)
     tokenizer = tokenizer_class.from_pretrained(args.tokenizer_name if args.tokenizer_name else args.model_name_or_path,
                                                 do_lower_case=args.do_lower_case,
-                                                cache_dir=args.cache_dir if args.cache_dir else None)
+                                                cache_dir=args.cache_dir if args.cache_dir else None,
+                                                xla_device=True)
     model = model_class.from_pretrained(args.model_name_or_path,
                                         from_tf=bool('.ckpt' in args.model_name_or_path),
                                         config=config,
-                                        cache_dir=args.cache_dir if args.cache_dir else None)
+                                        cache_dir=args.cache_dir if args.cache_dir else None,
```

Comment:
isn’t args.cache_dir None if not args.cache_dir?

Reply:
I think what they want to not have is stuff like empty strings. This comes from upstream code that was branched.